### PR TITLE
Working color picker

### PR DESCRIPTION
**Pull request by moxvallix**

### CODE REVIEW

**Review by nicemicro — CHANGES_REQUESTED** [review]
I don't think the switch to a signal is a good idea. Especially because skeletons will be used in a lot of places, and we only want the one specific skeleton in this case to change its appearance (the one that is in the menu).
As we know exactly what nodes the change affects, signals are not the best idea imho.

**Review by nicemicro — APPROVED** [review]
good.

**Review by TheSecondReal0 — CHANGES_REQUESTED** [review]
Please merge [this PR into your fork](https://github.com/moxvallix/opensus/pull/1), it makes it so that you can hold down click on a color picker to get constant updates and a color pointer to show what color is currently selected. 



Also this doesn't completely follow the style guide, but it doesn't have to be fixed here if you're planning on fixing it in another PR before merging into main

**Review by TheSecondReal0 — APPROVED** [review]

File: src/autoloads/appearance.gd [hunk 1]
```diff
@@ -2,6 +2,8 @@ extends Node
 
 onready var Resources = get_node("/root/Resources")
 
+signal applyConfig # Apply config for menu character
```
Comment (nicemicro) [inline src/autoloads/appearance.gd:5]: I'd rename this signal to something that specifies that the character customization has been accepted in the appearance screen.

**Review by nicemicro — CHANGES_REQUESTED** [review]
Color picker works as intended.
Rename the signal please to something more specific.

File: src/autoloads/appearance.gd (hunk 1 above)
Comment (moxvallix) [reply src/autoloads/appearance.gd:5]: I'll change this in the other PR

**Pull request merged**